main.go

```diff
@@ -56,16 +56,17 @@ func execWithCredentials(argv []string, creds *credentials) error {
 		return err
 	}
 
+	os.Setenv("AWS_ACCESS_KEY_ID", creds.AccessKeyID)
+	os.Setenv("AWS_SECRET_ACCESS_KEY", creds.SecretAccessKey)
+	os.Setenv("AWS_SESSION_TOKEN", creds.SessionToken)
+	os.Setenv("AWS_SECURITY_TOKEN", creds.SessionToken)
+
 	env := os.Environ()
-	env = append(env, fmt.Sprintf("AWS_ACCESS_KEY_ID=%s", creds.AccessKeyId))
-	env = append(env, fmt.Sprintf("AWS_SECRET_ACCESS_KEY=%s", creds.SecretAccessKey))
-	env = append(env, fmt.Sprintf("AWS_SESSION_TOKEN=%s", creds.SessionToken))
-	env = append(env, fmt.Sprintf("AWS_SECURITY_TOKEN=%s", creds.SessionToken))
 	return syscall.Exec(argv0, argv, env)
 }
 
 type credentials struct {
-	AccessKeyId     string
+	AccessKeyID     string
```

Comment:
I'd take more credit if it wasn't my editor yelling at me everytime I open someone elses go :)